drivers/frequency/admv1013/admv1013.c

```diff
@@ -39,6 +39,9 @@
 #include "no_os_error.h"
 #include "no_os_alloc.h"
 
+#define MIXER_GATE_0_to_1_8_V(x)		((2389 * x/ 1000000 + 8100) / 100)
```

Comment:
if doing the change, i'd also uppercase the "_to" -> "_TO_"

Reply:
will do